packages/eslint-config-custom/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+    extends: ["next", "turbo", "prettier"],
+    rules: {
+      "@next/next/no-html-link-for-pages": "off",
```

Comment:
😬 , we are not using this config...seems like an example config from the turbo repo?

Reply:
also, it doesn't seem like we have setup eslint in the photos app? I can just see the package in package.json but no usage of it.

Reply:
our eslint config https://github.com/httpjamesm/photos-web/blob/bc5c1c7f162da1dd0d970dde0d7160f030d754c8/.eslintrc.json